guides/getting-started.md

```diff
@@ -82,16 +82,15 @@ import {MatButtonModule, MatCheckboxModule} from '@angular/material';
 export class PizzaPartyAppModule { }
 ```
 
-Alternatively, you can create a separate NgModule that imports all of the
+Alternatively, you can create a separate NgModule that exports all of the
```

Comment:
Right. But personally, especially for newcomers, it might be better to also leave the `imports` statement as well. Usually where this importing/exporting happens is your `SharedModule`, where you most probably have also a bunch  of other stuff that you'd like to re-export + you might even have shared components there, using things from Angular Material, and thus the `import` would be required again.

So for simplicity, I'd nearly suggest changing it slightly to

"Alternatively, you can create a separate NgModule that imports and then re-exports all of the Anguar Material components that you will use in your application. By exporting them again, other modules can simply include our `MyOwnCustomMaterialModule` wherever Material components are needed and automatically get all of the exported Material modules. A good place for importing/exporting the application-wide Material modules is the [SharedModule](https://angular.io/guide/ngmodule-faq#sharedmodule)."

Something like this.

Reply:
I couldn't have said this any better.